travis/deploy-gh-pages.sh

```diff
@@ -11,7 +11,8 @@ then
 else
   export VERSION="${TRAVIS_BRANCH}"
 fi
-docker run --volume $(pwd)/docs:/home/node/apiary:Z giovannicuriel/aglio -i /home/node/apiary/api.apib -o - > ./apiary_${VERSION}.html
+
+node_modules/.bin/aglio -i api.apib -o apiary_${VERSION}.html
```

Comment:
Last time we measured, it took almost 3 minutes just to install aglio. Did it improved its install time?

Reply:
Increased 2 min.
But otherwise it was not working.
It generated the file, but without content.

Reply:
Maybe we need to find another way, It could be how you did.